tests/scrape/test_scrape.py

```diff
@@ -171,21 +174,57 @@ def _create_pdf_files(self, dir, meeting_dict):
                 pdf_file = year_dir / pdf_name
                 pdf_file.touch(exist_ok=True)
                 assert pdf_file.exists()
+                files.append(pdf_file)
+        return files
 
     def test_no_missing_pdfs(self, pdf_dir):
         """Tests that the function returns nothing when all pdfs are present"""
         # setup
-        self._create_pdf_files(pdf_dir, MEETING_LINKS)
+        pdf_files = self._create_pdf_files(pdf_dir, MEETING_LINKS)
+        for file in pdf_files:
```

Comment:
this for loop that asserts file existence seems redundant since it was already asserted to exist on line 176 in the helper function. It's not broken, it just doesn't seem necessary.

Reply:
Something really strange seems to be happening, the code that you're reviewing doesn't seem to be consistent with the current version on my branch: see how the parent class for this function is `class TestCheckMissingPDFs` and not `TestCheckFileList` https://github.com/widal001/BOE_tabulator/blob/3c954dcc06dcac3554dc56c6eb9d238816cad496/tests/scrape/test_scrape.py#L164

Reply:
Good catch with the extra `file.exists()` though, I must have left that in when I was pulling out the functionality into the helper function